euphoria-hbase/src/test/java/cz/seznam/euphoria/hbase/HFileSinkTest.java

```diff
@@ -64,29 +65,15 @@ void loadIncrementalHFiles(Path path) {
       super.loadIncrementalHFiles(path);
     }
 
-  };
+  }
 
   @Rule
   public TemporaryFolder folder = new TemporaryFolder();
 
-  final TableName table = TableName.valueOf("test");
+  private final TableName table = TableName.valueOf("test");
 
-  List<String> loadedPaths;
-  Flow flow;
-
-  @Before
-  @Override
-  public void setUp() throws Exception {
-    super.setUp();
-    flow = Flow.create();
-    loadedPaths = new ArrayList<>();
-  }
-
-  @After
-  @Override
-  public void tearDown() throws IOException {
-    super.tearDown();
-  }
+  private final List<String> loadedPaths = new ArrayList<>();
+  private final Flow flow = Flow.create();
```

Comment:
I think the flow should be created per test.

Reply:
fixed